scripts/spec-check.py

```
@@ -0,0 +1,124 @@
+        ids.add(ac.get("id"))
+
+    # 3. blastRadius 元素非空
+    if not all(isinstance(x, str) and x.strip() for x in fm["blastRadius"]):
+        errs.append("blastRadius 含空元素")
+
+    # 4. 实现细节关键词
+    for pat, why in IMPL_PATTERNS:
+        if re.search(pat, text):
+            errs.append(f"含实现细节（{why}）——spec 只描述是什么/验收什么")
+
+    # 5. 注入扫描（只扫 spec 正文语义——模式命中即报，宁枉勿纵；
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)